netci.groovy

```diff
@@ -1442,6 +1442,11 @@ def static calculateBuildCommands(def newJob, def scenario, def branch, def isPR
                         buildCommands += "set __TestIntermediateDir=int&&build.cmd ${lowerConfiguration} ${arch} ${buildOpts}"
                     }
 
+                    // If it is a release build for windows, ensure PGO is used, else fail the build
```

Comment:
Agreed that we shouldn't enforce pgo for legacy backend, since we aren't collecting for it.